podman-wrapper.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/bash
```

Comment:
this only works in Linux. #!/bin/bash will support macOS too

Reply:
good to know!
done